tox.ini

```
@@ -4,7 +4,7 @@ per-file-ignores=__init__.py:F401,tests/*:S,D100,D104
 docstring-convention=all
 import-order-style=pycharm
 application_import_names=api,tests
-exclude=gunicorn.conf.py
+exclude=gunicorn.conf.py, alembic
```

I am not sure if we would want to exclude them? We can just fix the `script.mako` file to make it compliant with black and flake8

---

As far as I am concerned, migrations shouldn't be a problem of contributors, as they're automatically generated.
(That's why they're excluded in case of the older API.

---

I would like to keep black, flake8 can be ignored, see my previous comments.

---

> As far as I am concerned, migrations shouldn't be a problem of contributors, as they're automatically generated.
> (That's why they're excluded in case of the older API.)

As far as I recall, migrations in Alembic are not generated automatically, in contrast to the Django ORM - we need to build them manually. I _think_ there is some way to autogenerate them by diffing sqlalchemy models, but I haven't worked with it personally yet, and the Django autogenerate support is an entirely different level (I never had to write a Django migration manually). So this should be kept in the linting, yeah:

```suggestion
exclude=gunicorn.conf.py
```

---

The migrations committed here, were generated automatically.